addons/storyshots/storyshots-core/.eslintrc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   settings: {
     'import/core-modules': [
       '@storybook/angular',
+      '@storybook/ember',
```

Comment:
This isn't needed unless you add ember support in storyshots

Reply:
I tried to add support, but ember doesn't currently have jest support

Reply:
That's OK, please just remove this line for now